### PR TITLE
Migrate missing content

### PR DESCRIPTION
**Pull request by sebclrsn**
Update pykiso to 0.14.0 and re-update all files

### CODE REVIEW

File: examples/ext_lib/cc_example.py [hunk 1]
```diff
@@ -9,12 +9,10 @@
 
 """
 Virtual Communication channel for tests
-***************************************
+****************************************
```
Comment (Pog3k) [inline examples/ext_lib/cc_example.py:12]: 1 char to much ?
```suggestion
***************************************
```

File: src/pykiso/lib/auxiliaries/proxy_auxiliary.py [hunk 2]
```diff
@@ -121,7 +120,7 @@ def _init_trace(
             dir_path / t_name if dir_path.exists() else (Path() / t_name).resolve()
         )
 
-        # configure the file handler and create the trace file
+        # configure the file handler and create the tarce file
```
Comment (Pog3k) [inline src/pykiso/lib/auxiliaries/proxy_auxiliary.py:123]:
```suggestion
        # configure the file handler and create the trace file
```

**Review by Pog3k — APPROVED** [review]

File: examples/ext_lib/cc_example.py (hunk 1 above)
Comment (sebclrsn) [reply examples/ext_lib/cc_example.py]: Didn't see that one, fixed

File: src/pykiso/lib/auxiliaries/proxy_auxiliary.py (hunk 2 above)
Comment (sebclrsn) [reply src/pykiso/lib/auxiliaries/proxy_auxiliary.py]: Fixed

File: examples/templates/config_connectors.yaml [hunk 3]
```diff
@@ -254,7 +254,7 @@ connectors:
   visa_tcp_ip_channel:
     config:
       # serial communication port in use
-      ip_address: '127.0.0.1'
+      ip_address: 127.0.0.1
```
[the remark this answers is not in the document]
Comment (sebclrsn) [reply examples/templates/config_connectors.yaml]: Everything's reverted

File: examples/nested_yaml/flasher_config/flash_lauterbach.yaml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 config:
-  t32_exc_path: 'path/to/trace32.exe'
-  t32_config: '../path/to/config.t32'
-  t32_script_path: '../path/to/flash/script.cmm'
-  t32_api_path: 'path/to/trace32.dll'
+  t32_exc_path: 'C:/T32/bin/windows64/t32mppc.exe'
+  t32_config: '../test_suite_fdx_lauterbach/config_mc.t32'
+  t32_script_path: '../test_suite_fdx_lauterbach/BES3_FlashApplication_TAPP_Demo.cmm'
+  t32_api_path: 'C:/T32/demo/api/capi/dll/t32api.dll'
```
[the remark this answers is not in the document]
Comment (sebclrsn) [reply examples/nested_yaml/flasher_config/flash_lauterbach.yaml]: Done

File: examples/templates/config_connectors.yaml [hunk 5]
```diff
@@ -166,7 +166,7 @@ connectors:
       # id used for transmission
       remote_id: 0x123
       # iterable used to filter can id on reception
-      can_filters: [{"can_id": 0x123, "can_mask": 0xFFF, "extended": False}]
+      can_filters: [{"can_id": 0x600, "can_mask": 0xFFF, "extended": False}]
```
[the remark this answers is not in the document]
Comment (sebclrsn) [reply examples/templates/config_connectors.yaml]: Done

**Review by kakooloukia — APPROVED** [review]

**Pull request merged**